### PR TITLE
Update ci-fix.md: update Prettier check guidance

### PR DESCRIPTION
**Pull request by konrad-jamrozik**
This is a PR made by the Azure SDK Engineering System team.

This PR improves `prettier` troubleshooting guide (TSG) at https://aka.ms/ci-fix.

For context how this update came to be, see [this Teams discussion](https://teams.microsoft.com/l/message/19:0351f5f9404446e4b4fd4eaf2c27448d@thread.skype/1685746275186?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1685715519985&teamName=Azure%20SDK&channelName=API%20Spec%20Review&createdTime=1685746275186).

### CODE REVIEW

File: documentation/ci-fix.md [hunk 1]
```diff
@@ -49,6 +50,20 @@ Then please commit and push changes made by prettier.
...
+```
+[error] Cannot find module 'prettier/parser-babylon'
+[error] Require stack:
+[error] - azure-rest-api-specs-pr\scripts\prettier-swagger-plugin.js
+[error] - azure-rest-api-specs-pr\node_modules\prettier\index.js
+[error] - azure-rest-api-specs-pr\node_modules\prettier\cli.js
+[error] - azure-rest-api-specs-pr\node_modules\prettier\bin-prettier.js
+```
+
+then ensure you ran `npx tsc` from `main` branch and did not run `npx tsc` from any other branch, like `RPSaaSDev`.
```
Comment (weshaggard) [inline documentation/ci-fix.md:65]: If folks are working in another branch what do you expect them to do? I think we need to figure out how to make this work correctly in any of our primary branches.

Comment (konrad-jamrozik) [reply documentation/ci-fix.md:65]: I agree, but maybe we could update these instructions for now as a stop-gap solution?

Comment (konrad-jamrozik) [reply documentation/ci-fix.md]: I added clarification.

Comment (konrad-jamrozik) [reply documentation/ci-fix.md]: @weshaggard per our chat, you will take point on figuring out why the error is happening on the dev branch and override it to fix the npx tsc on it (see file diff for the error).

**Comment by weshaggard** [comment]
Closing this as I'm fixing the RPSaaS* branches to not need this extra information

**Pull request closed without merging**